packages/account/src/test-utils/setup-test-provider-and-wallets.ts

```diff
@@ -64,15 +64,33 @@ export async function setupTestProviderAndWallets({
     }
   );
 
-  const { cleanup, url } = await launchNode({
+  const launchNodeOptions = {
     loggingEnabled: false,
     ...nodeOptions,
     snapshotConfig: mergeDeepRight(
       defaultSnapshotConfigs,
       walletsConfig.apply(nodeOptions?.snapshotConfig)
     ),
     port: '0',
-  });
+  };
+
+  let cleanup: () => void;
+  let url: string;
+  if (process.env.LAUNCH_NODE_SERVER_PORT) {
```

Comment:
I believe the utility should work purely from arguments.

Env variables are counterintuitive and shouldn't be used in anything related to `testNodeLaunch`.

Reply:
Moved to argument in https://github.com/FuelLabs/fuels-ts/pull/2378/commits/c6005bee43e6cc1a76107a13cbafeec54f946dbe. I didn't document the new `launchNodeServerPort` argument because the infrastructure added in the PR is still internal. When we decide to make it public, we can add proper documentation and showcase the bigger picture properly.